### ISSUE TITLE
"Clarify Purpose of `Program`/`Programs` Classes with More Semantically Accurate Naming"

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/lints#501]
Current naming of `Program.java` (single XMIR) and `Programs.java` (multiple XMIRs) is confusing and misleading.  Usually, a program consists of many source files, whereas programs refers to several projects. Now, as I see it from the source code, we have absolutely different mapping. 

```java
/**
 * A single XMIR program to analyze.
 *
 * @see <a href="https://news.eolang.org/2022-11-25-xmir-guide.html">XMIR</a>
 * @since 0.1.0
 */
public final class Program {

    /**
     * The XMIR program to analyze.
     */
    private final XML xmir;
}
```

It's not a single "XMIR program". It's a single XMIR file.
Suggest renaming to `Source.java` and `Program.java` respectively for clearer semantics (`Program`->`Source.java`, `Programs` -> `Program`). I'm not insist on these concrete names, but it's better to rename them.

**Comment by volodya-lombrozo** [comment]
@yegor256 @h1alexbel What do you think?

**Comment by yegor256** [comment]
@volodya-lombrozo good idea. Maybe `Object` and `Program`?

**Comment by 0crat** [comment]
@volodya-lombrozo Great job on reporting that new bug! 🐛 As per our policy, you've earned **8** points for your keen eye. Your contribution is invaluable in enhancing our project's quality. Keep an eye out for anything else that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) up to par, and [don't hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Remember, every bug you find helps us improve! Your current balance stands at [+71](https://www.eolang.org/zerocracy/objectionary-vitals.html) points. Keep up the excellent work!

**Comment by volodya-lombrozo** [comment]
@yegor256 "Object" is a special Java class which we use rather often. Maybe something similar? EoObject?

**Comment by h1alexbel** [comment]
@volodya-lombrozo @yegor256 personally, I like `Source.java` and `Program.java`

**Comment by yegor256** [comment]
@volodya-lombrozo @h1alexbel `Source` works for me too
